sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py

```diff
@@ -180,11 +202,19 @@ async def run(args):
             "track_last_enqueued_event_properties": args.track_last_enqueued_event_properties,
             "starting_position": starting_position
         }
+        if args.max_batch_size:
+            kwargs_dict["max_batch_size"] = args.max_batch_size
+            if args.max_wait_time:
+                kwargs_dict["max_wait_time"] = args.max_wait_time
         if args.parallel_recv_cnt and args.parallel_recv_cnt > 1:
             clients = [create_client(args) for _ in range(args.parallel_recv_cnt)]
             tasks = [
                 asyncio.ensure_future(
-                    clients[i].receive(
+                    clients[i].receive_batch(
+                        on_event_batch_received,
+                        prefetch=args.link_credit,
+                        max_batch_size=args.max_batch_size
+                    ) if args.max_batch_size else clients[0].receive_batch(
```

Comment:
according to the description: "Call EventHubConsumerClient.receive_batch() if not 0, otherwise call receive()"

it should be
```
else clients[0].receive()
```

Reply:
yes, this is a copy code error. Corrected